Gruntfile.js

```diff
@@ -226,4 +226,5 @@
 		grunt.registerTask( "pushBetaFW", [ "compress:makeFW", "shell:updateBetaUI", "clean:pushFW" ] );
 		grunt.registerTask( "build", [ "default", "shell:symres", "pushFW", "clean:symres" ] );
 		grunt.registerTask( "bump", [ "default", "replace:about", "replace:cordova", "replace:manifests", "shell:pushBump" ] );
+		grunt.registerTask( "bump-version", [ "default", "replace:about", "replace:cordova", "replace:manifests" ] );
```

Comment:
It would be great to have this task run when we merge developer PRs into the main repo! We can do that with a github action/workflow and be sure that our version numbers are always bumped.

Reply:
Sounds like a great idea, alternatively we could use GitVersion as well.